Packs/CheckpointFirewall/Playbooks/Checkpoint_-_Block_IP_-_Append_Group.yml

```diff
@@ -0,0 +1,1160 @@
+      script: '|||checkpoint-group-get'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "9"
+    scriptarguments:
+      identifier:
+        simple: ${inputs.group_name}
+      session_id:
+        simple: ${CheckPoint.Login.session-id}
+    continueonerror: true
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 220,
+          "y": 1870
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "35":
+    id: "35"
+    taskid: 6bdf7795-cd12-4ece-825f-701cc62b6457
+    type: regular
+    task:
+      id: 6bdf7795-cd12-4ece-825f-701cc62b6457
+      version: -1
+      name: Get current group members
+      description: |-
+        Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-2/cortex-xsoar-admin/playbooks/automations.html
+      scriptName: SetAndHandleEmpty
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "30"
+    scriptarguments:
+      key:
+        simple: currentMembers
+      value:
+        complex:
+          root: CheckPoint.Group
+          filters:
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: CheckPoint.Group.name
+                iscontext: true
+              right:
+                value:
+                  simple: inputs.group_name
+                iscontext: true
+              ignorecase: true
+          accessor: members.member-name
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 220,
+          "y": 1040
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 3975,
+        "width": 1550,
+        "x": -330,
+        "y": -900
+      }
+    }
+  }
+inputs:
+- key: IP
+  value: {}
+  required: true
+  description: Array of malicious IPs to block.
+  playbookInputQuery:
+- key: install_policy
+  value:
+    simple: "False"
+  required: true
+  description: Input True / False for playbook to continue install policy process
+    for checkpoint Firewall.
+  playbookInputQuery:
+- key: policy_package
+  value:
+    simple: Standard
+  required: true
+  description: The name of the policy package to be installed.
+  playbookInputQuery:
+- key: block_IP_error_handling
```

Comment:
The block_IP_error_handling and checkpoint_error_handling seem to be the same
Please explain the difference better.

Reply:
You are right! they are the same - that's why in the playbook I changed the way those errors were handled. Instead of 2 breaking points, I turned it to one (with OR operator between them) - and that's how I for rid of unnecessary tasks.

Reply:
![image](https://user-images.githubusercontent.com/88268646/147732595-562702db-05a4-47a2-9777-957c810db971.png)
Forgot to mention that I didn't delete this input as it may impact existing customers (BC).